src/main/java/com/quirko/logic/bricks/MediumBricks.java

```diff
@@ -0,0 +1,21 @@
+package com.quirko.logic.bricks;
+
+import java.util.List;
+import java.util.ArrayList;
+    
+public class MediumBricks {
+    
+    public List<Brick> brickList;
+
+    public MediumBricks(){
+        brickList = new ArrayList<>();
+        brickList.add(new IBrick());
```

Comment:
IBrick hem burada geçiyor hem de EasyBricks te geçiyor. Bu da bana bu classlarda tasarım hatası olduğunu söylüyor.

Senin herhalde yapmak istediğin farklı brick konfigürasyonları tanımlamak.

Öncelikle Brick typelarını kolay orta zor diye ayır.
Sonra ikinci levelde EasyConfiguration, MediumConfiguration, HardConfiguration diye classlar içerisinde kolay orta zor brick typelarından listeler oluşturup dönsün.

Şu anda iki farklı concern bir arada karıştırılmış. Bu concernleri ayıralım.